doc/cephfs/administration.rst

```diff
@@ -6,6 +6,9 @@ CephFS Administrative commands
 File Systems
 ------------
 
+.. note:: The names of the file systems, metadata pools, and data pools can
+          only have characters in the set [a-zA-Z0-9\_-.].
```

Comment:
Note: backslash is used to escape underscore, which is used to create hyper links. It doesn't appear in the doc render.